safe_transaction_service/tokens/services/price_service.py

```diff
@@ -212,6 +215,11 @@ def get_native_coin_usd_price(self) -> float:
             return self.coingecko_client.get_gather_usd_price()
         elif self.ethereum_network == EthereumNetwork.AVALANCHE:
             return self.get_avalanche_usd_price()
+        elif self.ethereum_network in (
```

Comment:
Add a test for your 2 networks by mocking the `BinanceClient`: https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/tokens/tests/test_price_service.py#L105